worker/config.yaml

```diff
@@ -0,0 +1,59 @@
+# API server listen address
+# Defauls to: ":8080" which will listen on all avalable interfaces.
+# Overwrite with environment variable: WORKFLOWS_LISTEN
+
+listen: :8080
+
+# Mongodb connection string
+# Defaults to: "mongodb://localhost"
+# Overwrite with environment variable: WORKFLOWS_MONGO_URL
+
+mongo-url: mongodb://mongo-workflows:27017
+
+# Mongodb database name
+# Defaults to: "workflows"
+# Overwrite with environment variable: WORKFLOWS_MONGO_DBNAME
+
+mongo-dbname: workflows
+
+# Enable SSL for mongo connections
+# Defaults to: false
+# Overwrite with environment variable: WORKFLOWS_MONGO_SSL
+
+# mongo_ssl: false
+
+# SkipVerify controls whether a mongo client verifies the
+# server's certificate chain and host name.
+# If InsecureSkipVerify is true, accepts any certificate
+# presented by the server and any host name in that certificate.
+# Defaults to: false
+# Overwrite with environment variable: WORKFLOWS_MONGO_SSL_SKIPVERIFY
+
+# mongo_ssl_skipverify: false
+
+# Mongodb username
+# Overwrites username set in connection string.
+# Defaults to: none
+# Overwrite with environment variable: WORKFLOWS_MONGO_USERNAME
+
+# mongo_username: user
+
+# Mongodb password
+# Overwrites password set in connection string.
+# Defaults to: none
+# Overwrite with environment variable: WORKFLOWS_MONGO_PASSWORD
+
+# mongo_password: secret
+
+# Workflows path
+# The path containing the workflows definitions
+# Defaults to: "./etc"
+# Overwrite with environment variable: WORKFLOWS_WORKFLOWS_PATH
+workflows_path: "./etc"
```

Comment:
this is really /etc and works because CWD of the process is / wouldnt it be better to use `"/etc"` here? since we are copying the definitions to /etc [here](https://github.com/mendersoftware/workflows/pull/17/commits/5b42a3e594189db677aa759df6097a5943ed0def#diff-3e6294f438c3be45991ac029ece67fd1R15)

Reply:
You are right, I expected it to be overridden by env variable but it makes no sense, as we are building the docker image for the worker let's add a config file which match with the paths we are using.

Force-pushed with the fix.